### PR TITLE
Fixes in horizontal_text_detection documentation

### PR DESCRIPTION
**Pull request by morkovka1337**
Updated documentation correspondingly to the current folder structure. Also added information about venv activation, though it should be discussed if it is really needed.
Changed python3 to python since this would be executed inside venv anyway.
Added new argument `--root` to the doc since it is required in `create_dataset.py` script

### CODE REVIEW

File: pytorch_toolkit/object_detection/model_templates/horizontal-text-detection/readme.md [hunk 1]
```diff
@@ -8,10 +8,15 @@ Model that is able to detect more or less horizontal text with high speed on CPU
 
 ## Training pipeline
 
-### 0. Change a directory in your terminal to object_detection.
+### 0. Change a directory in your terminal to object_detection and activate venv.
```
Comment (Ilya-Krylov) [inline pytorch_toolkit/object_detection/model_templates/horizontal-text-detection/readme.md:11]: Regarding this change: activating of venv is mentioned here: https://github.com/openvinotoolkit/training_extensions/tree/develop/pytorch_toolkit/object_detection/readme.md

If you would like to apply your change please do it for all templates

File: pytorch_toolkit/object_detection/model_templates/horizontal-text-detection/readme.md [hunk 2]
```diff
@@ -46,17 +51,19 @@ Convert it to format that is used internally and split to the train and test par
 
 * Training annotation
 ```bash
-python3 ./model_templates/horizontal-text-detection/tools/create_dataset.py \
-    --config ./model_templates/horizontal-text-detection/datasets/dataset_train.json \
-    --output ${DATA_DIR}/text-dataset/IC13TRAIN_IC15_IC17_IC19_MSRATD500_COCOTEXT.json
+python ./model_templates/horizontal-text-detection/tools/create_dataset.py \
+    --config ./model_templates/horizontal-text-detection/tools/datasets/dataset_train.json \
+    --output ${DATA_DIR}/text-dataset/IC13TRAIN_IC15_IC17_IC19_MSRATD500_COCOTEXT.json \
+    --root ./
```
Comment (Ilya-Krylov) [inline pytorch_toolkit/object_detection/model_templates/horizontal-text-detection/readme.md:57]: I guess that root should be `${DATA_DIR}/text-dataset/`.

File: pytorch_toolkit/object_detection/model_templates/horizontal-text-detection/readme.md [hunk 3]
```diff
@@ -46,17 +51,19 @@ Convert it to format that is used internally and split to the train and test par
...
 ```
 * Testing annotation
 ```bash
-python3 ./model_templates/horizontal-text-detection/tools/create_dataset.py \
-    --config ./model_templates/horizontal-text-detection/datasets/dataset_test.json \
-    --output ${DATA_DIR}/text-dataset/IC13TEST.json
+python ./model_templates/horizontal-text-detection/tools/create_dataset.py \
+    --config ./model_templates/horizontal-text-detection/tools/datasets/dataset_test.json \
+    --output ${DATA_DIR}/text-dataset/IC13TEST.json \
+    --root ./
```
Comment (Ilya-Krylov) [inline pytorch_toolkit/object_detection/model_templates/horizontal-text-detection/readme.md:66]: ditto

Comment (Ilya-Krylov) [inline pytorch_toolkit/object_detection/model_templates/horizontal-text-detection/readme.md:66]: please update alphanumeric-text-spotting page as well

**Review by Ilya-Krylov — CHANGES_REQUESTED** [review]

File: pytorch_toolkit/object_detection/model_templates/horizontal-text-detection/readme.md (hunk 2 above)
Comment (morkovka1337) [reply pytorch_toolkit/object_detection/model_templates/horizontal-text-detection/readme.md]: Fixed

File: pytorch_toolkit/object_detection/model_templates/horizontal-text-detection/readme.md (hunk 3 above)
Comment (morkovka1337) [reply pytorch_toolkit/object_detection/model_templates/horizontal-text-detection/readme.md]: Fixed

File: pytorch_toolkit/object_detection/model_templates/horizontal-text-detection/readme.md (hunk 1 above)
Comment (morkovka1337) [reply pytorch_toolkit/object_detection/model_templates/horizontal-text-detection/readme.md:11]: Added info about virtual environment activation to other templates

File: pytorch_toolkit/object_detection/model_templates/horizontal-text-detection/readme.md (hunk 3 above)
Comment (morkovka1337) [reply pytorch_toolkit/object_detection/model_templates/horizontal-text-detection/readme.md]: Updated

**Comment by Ilya-Krylov** [comment]
@morkovka1337 where does `venv` come from to domain folder? It is worth to mention this as well.

**Review by GalyaZalesskaya — APPROVED** [review]

**Review by Ilya-Krylov — APPROVED** [review]

**Comment by Ilya-Krylov** [comment]
run tests

**Pull request merged**